vite.config.ts

```diff
@@ -10,7 +10,7 @@ export default defineConfig({
     css: true,
     coverage: {
       provider: "istanbul",
-      include: ["src/components"],
+      include: ["src/components", "src/utils.ts"],
```

Comment:
`mixins` dir too? Is there an easier way to do to this, maybe include all `src` but exclude the files we don't want to test?

Reply:
Agreed, feels safer to include everything be default, then exclude selectively, otherwise to easy to forget to include new code.

Reply:
done, am just including src now